lib/tasks/rake_config.rb

```diff
@@ -6,16 +6,30 @@ def context
 
     attr_writer :context
 
-    def config
+    def config(is_cc_local_worker: false)
```

Comment:
You don't need this `is_cc_local_worker` parameter but can use the `context` instead (that is either `:api` for local workers or `:worker` for generic workers.